meta-ibm/meta-witherspoon/recipes-phosphor/fans/phosphor-fan-control-fan-config/mowgli/fans.yaml

```diff
@@ -0,0 +1,31 @@
+fans:
+  - inventory: /system/chassis/motherboard/fan0
+    cooling_zone: 0
+    sensors:
+      - fan0_0
+      - fan0_1
```

Comment:
Do you have separate PWM control lines to each rotor within a fan? If yes, then this is correct...otherwise, you only need to list the fan sensors that have the target interface here.

Reply:
Yes, we only have fan*_0 with target interface, so I deleted fan*_1 sensor.